app/questionnaire/questionnaire_store_updater.py

```diff
@@ -149,10 +149,10 @@ def remove_primary_person(self, list_name: str) -> None:
         """
         list_item_id = self._list_store[list_name].primary_person
         if list_item_id:
-            self.remove_list_item_and_answers(list_name, list_item_id)
+            self.remove_list_item_from_all_stores(list_name, list_item_id)
 
-    def remove_list_item_and_answers(self, list_name: str, list_item_id: str) -> None:
-        """Remove answers from the answer store and update the list store to remove it.
+    def remove_list_item_from_all_stores(self, list_name: str, list_item_id: str) -> None:
```

Comment:
Minor, all store isn't true?. We will soon has sds store. Maybe just `remove_list_item_data`?

Reply:
Done 👌